browser/src/view/components/Navbar/NavigationMenu.js

```diff
@@ -1,31 +1,32 @@
 import React from 'react';
-import { FiHome, FiLayers, FiSettings, FiLogOut, FiCircle } from 'react-icons/fi';
-import './NavigationMenu.css';
+import PropTypes from 'prop-types';
+import './NavigationMenu.scss';
 
-export const NavigationMenu = () => {
+export const NavigationMenu = ({ links }) => {
   return (
-    <>
-      <div className='navbar'>
-        <div className='nav-top'>
-          <div className='icon-container'>
-            <FiCircle className='nav-icon logo' />
-          </div>
-
-          <div className='icon-container'>
-            <FiHome className='nav-icon' />
-          </div>
-
-          <div className='icon-container'>
-            <FiLayers className='nav-icon' />
-          </div>
-        </div>
-        <div className='nav-bottom'>
-          <div className='icon-container'>
-            <FiSettings className='nav-icon' />
-          </div>
-          <div className='avatar'></div>
-        </div>
-      </div>
-    </>
+    <nav className='navbar'>
+      <ul className='nav-section-one'>
+        {links.map(link => {
+          if (link.name !== 'Settings') {
```

Comment:
We generally want to keep logic out of templates.  Stateful (state, code paths) templates can cause issues because  you end up having state living in multiple places.  Refactoring becomes difficult given that templates are pretty volatile (change often).

In this case, settings seems like it doesn't need to be provided at all.

Reply:
That makes sense. I forgot to even think about this as a template. I'll rework it to remove the logic and statefulness here.